### PR TITLE
update(mithril): Mithril adapter

### PR DESCRIPTION
**Pull request by maicol07**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Mithril_ | _https://github.com/maicol07/inertia-mithril_ |

_The updated Mithril adapter for Inertia_

- Updated the link with the new package, updated for latest Inertia
- Labeled the old Mithril adapter as "up to Inertia 0.1)

---

- [x] My item is in the right category
- [ ] My item has been added to the bottom of the list
- [x] My item's name and description respects the conventions of the list
- [x] I have read and followed the [contributing guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,8 @@
 - 💜 [Vue.js](https://github.com/inertiajs/inertia/tree/master/packages/inertia-vue)
 - 💜 [React](https://github.com/inertiajs/inertia/tree/master/packages/inertia-react)
 - 💜 [Svelte](https://github.com/inertiajs/inertia/tree/master/packages/inertia-svelte)
-- [Mithril](https://github.com/tbreuss/inertia-mithril)
+- [Mithril](https://github.com/maicol07/inertia-mithril)
+- [Mithril (up to Inertia 0.1)](https://github.com/tbreuss/inertia-mithril)
```
Comment (innocenzi) [inline README.md:59]:
```suggestion
```

You can remove this one 👍

**Review by innocenzi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maicol07) [reply README.md:59]: :thinking: I don't know if my adapter works with oldest version, but if the list only allows latest Inertia versions, then I can remove it.

Comment (innocenzi) [reply README.md:59]: The older Inertia versions are not supposed to be used any longer - if someone is starting a project and is looking for a Mithril adapter, they'll use the latest versions. So yes you can remove it 👍

Comment (maicol07) [reply README.md]: Done!

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Thanks!

**Pull request merged**